mongo/doc.go

```diff
@@ -9,14 +9,12 @@
 // Package mongo provides a MongoDB Driver API for Go.
 //
 // Basic usage of the driver starts with creating a Client from a connection
-// string. To do so, call the NewClient and Connect functions:
+// string. To do so, call Connect:
```

Comment:
Just curious, why are we encouraging `Connect()` over `NewClient()`?

Reply:
Users can get confused when there are two ways to do something and tend to prefer conciseness. There isn't really a situation where they need to use NewClient() so its easier if we don't mention it.